packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -82,39 +84,39 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
       (this._hasExpandButton ? _indentWithExpandButton : _indentNoExpandButton)).toString(10) + 'px';
 
     return (
-        <a
-          className={ css('ms-Nav-link') }
-          style={ { [isRtl ? 'paddingRight' : 'paddingLeft'] : paddingBefore } }
-          href={ link.url || 'javascript:' }
-          onClick={ this._onNavAnchorLinkClicked.bind(this, link) }
-          aria-label={ link.ariaLabel }
-          title={ link.title || link.name }
-          target={ link.target }
+      <a
+        className={ css('ms-Nav-link') }
+        style={ { [isRtl ? 'paddingRight' : 'paddingLeft']: paddingBefore } }
+        href={ link.url || 'javascript:' }
+        onClick={ this._onNavAnchorLinkClicked.bind(this, link) }
+        aria-label={ link.ariaLabel }
+        title={ link.title || link.name }
+        target={ link.target }
         >
-         { link.iconClassName && <i className={ css('ms-Icon', 'ms-Nav-IconLink', link.iconClassName) }></i> }
-         { this.props.onRenderLink(link)}
-        </a>
+        { link.iconClassName && <i className={ css('ms-Icon', 'ms-Nav-IconLink', link.iconClassName) }></i> }
+        { this.props.onRenderLink(link) }
+      </a>
     );
   }
 
   private _renderButtonLink(link: INavLink, linkIndex: number) {
     return (
       <Button
         className={ css('ms-Nav-link ms-Nav-linkButton', { 'isOnExpanded': this._hasExpandButton }) }
-          buttonType={ ButtonType.command }
-          icon={ link.icon }
-          description={ link.title || link.name }
-          onClick={ this._onNavButtonLinkClicked.bind(this, link) }>
-          { link.name }
+        buttonType={ ButtonType.command }
+        icon={ link.icon }
+        description={ link.title || link.name }
```

Comment:
this is something messed up I did not change this or add this at all.  it is the same code.  My only change in this dpk is about 4 lines but somehow due to white spaces change merged from master, it show up this many changes.